docs/user-guide/logging/stackdriver.md

```diff
@@ -20,149 +24,125 @@ fluentd-cloud-logging-kubernetes-node-0f64     1/1       Running   0          32
+System components have different resource type, `compute`, named
+`GCE VM Instance` in the interface. Log names for system components are fixed.
+For a GKE node, every log entry from a system component has one the following
+log names:
 
-![Both Logs](/images/docs/all-lines.png)
+* docker
+* kubelet
+* kube-proxy
 
-Note the first container counted to 108 and then it was terminated. When the next container image restarted the counting process resumed from 0. Similarly if we deleted the pod and restarted it we would capture the logs for all instances of the containers in the pod whenever the pod was running.
-
- Logs ingested into Stackdriver Logging may be exported to various other destinations including [Google Cloud Storage](https://cloud.google.com/storage/) buckets and [BigQuery](https://cloud.google.com/bigquery/). Use the Exports tab in the Cloud Logging console to specify where logs should be streamed to. You can also follow this link to the
- [settings tab](https://pantheon.corp.google.com/project/_/logs/settings).
-
- We could query the ingested logs from BigQuery using the SQL query which reports the counter log lines showing the newest lines first:
-
-```shell
-SELECT metadata.timestamp, structPayload.log
-FROM [mylogs.kubernetes_counter_default_count_20150611]
-ORDER BY metadata.timestamp DESC
-```
-
-Here is some sample output:
-
-![BigQuery](/images/docs/bigquery-logging.png)
-
-We could also fetch the logs from Google Cloud Storage buckets to our desktop or laptop and then search them locally. The following command fetches logs for the counter pod running in a cluster which is itself in a Compute Engine project called `myproject`. Only logs for the date 2015-06-11 are fetched.
+You can learn more about viewing logs on [the dedicated Stackdriver page](https://cloud.google.com/logging/docs/view/logs_viewer).
 
+One of the possible ways to view logs in using [gcloud](https://cloud.google.com/sdk/gcloud/)
+– command line tool from a Google Cloud SDK. It uses Stackdriver Logging
+[filtering syntax](https://cloud.google.com/logging/docs/view/advanced_filters)
+to query specific logs. For example, you can run the following command
 
 ```shell
-$ gsutil -m cp -r gs://myproject/kubernetes.counter_default_count/2015/06/11 .
-```
-
-Now we can run queries over the ingested logs. The example below uses the [jq](http://stedolan.github.io/jq/) program to extract just the log lines.
-
-```shell
-$ cat 21\:00\:00_21\:59\:59_S0.json | jq '.structPayload.log'
-"0: Thu Jun 11 21:39:38 UTC 2015\n"
-"1: Thu Jun 11 21:39:39 UTC 2015\n"
-"2: Thu Jun 11 21:39:40 UTC 2015\n"
-"3: Thu Jun 11 21:39:41 UTC 2015\n"
-"4: Thu Jun 11 21:39:42 UTC 2015\n"
-"5: Thu Jun 11 21:39:43 UTC 2015\n"
-"6: Thu Jun 11 21:39:44 UTC 2015\n"
-"7: Thu Jun 11 21:39:45 UTC 2015\n"
+$ gcloud beta logging read 'logName="projects/$YOUR_PROJECT_ID/logs/count"' --format json | jq '.[].textPayload'
```

Comment:
@sharbison3 Thanks a lot for the clarification!

On the "Files changes" page you can click on the little plus sign near the line number (it appears as you hover over the line), write a comment and submit a review.